styleguide/components/41-organisms/timeline/timeline.twig

```diff
@@ -172,6 +172,101 @@
         </div>
       </dd>
     </div>
+    <div class="timeline-slot">
+      <dt>1,4 till 1,6 milion years ago</dt>
+      <dd>
+        <div class="timeline-slot-header">
+          <h3 class="timeline-slot-title"><button aria-expanded="true" aria-controls="timeline-slot-content--7" class="accordion--button">Title</button></h3>
+          <span class="timeline-slot-date">On a thursday</span>
+          <div class="image-gallery-wrapper">
+            {% include '@image-gallery' with {
+              "src": "https://loremflickr.com/704/440/ghent?random=14",
```

Comment:
Is het niet beter om hier ook de 800x500 rule te hanteren? Tevens in organisms zou ik geen loremflickr gebruiken omdat je hierdoor de ratio docs mist.

Reply:
Aangepast.